### ISSUE TITLE
eval and test use the same dataset?

### ISSUE THREAD

**Issue by ksmdnl** [amdegroot/ssd.pytorch#565]
I'm still trying to figure it if my assumption that the same dataset for evaluation and testing is being used. 
Did I miss anything there somewhere?
Head ups will be appreciated!

**Comment by smallsun1122** [comment]
I now have similar doubts, the training process seems to use the 2007VOC and 2012VOC, why the eval.py used in the 2007VOC?

**Comment by ksmdnl** [comment]
I suppose they were benchmarking or trying to reproduce the original implementation on caffe, which trained the model on trainval set and evaluated it on the test set.
I did train my model on both combinations and got, as expected, slightly different performance.
